### DISCUSSION TITLE
Need some UI elements for calendar event

### DISCUSSION THREAD

**Discussion by dianaspetcu** [illacloud/illa-builder#2162] [English Support]
Hi team, 
Can you make some futures for event calendar UI?
I make a list for some features that I need:

•	Different text styles in order to make the design friendlier; 
•	We don' t have the proper year view for the 12 months in a year; (all months in one single page)
•	We don' t have the possibility to change the text color according to the status of the timesheets( red- #C4192D , black-080808, green- #567B57, beige- #F6CFB2); 
•	We can’ t integrate in the calendar the errors regarding the 30 min break after 6h of work; 
•	We can' t integrate in the calendar a text that shows how many hours have been completed from the total of 8h per day; 
•	We can' t color the name of the day in red when all the working hours haven' t been completed; 
•	We can't block weekend days in the calendar, so that no registrations can be made on them; 
•	We can't edit the calendar so that the week starts with Monday; 
•	We can' t have a copy button in order to duplicate an event in the calendar; 
•	We can't put in an event box an add button and a delete button to delete or add an event; 
•	We can' t include icons on the buttons in order to suggest their use ; 
•	We can' t setup the start hour for a day to be from 8 am not 00 am.

**Comment by wenzhuo2022** [comment]
Hello, the features you mentioned are also very valuable to us, and we will optimize them within two months.

**Reply by dianaspetcu** [reply]
Thank you very much for the update :)
